controllers/constant/constants.go

```diff
@@ -25,6 +25,11 @@ const (
 	DefaultRestartLabelsValue       = "projected"
 	NamespaceScopeServiceAccount    = "ibm-namespace-scope-operator"
 	InjectorMark                    = "nss.operator.ibm.com/managed-operators"
+	WebhookMark                     = "nss.operator.ibm.com/patch-webhook"
 	NamespaceScopeConfigmapLabelKey = "namespace-scope-configmap"
 	NamespaceScopeRuntimePrefix     = "nss-runtime-managed-role-from-"
+	OlmOwnerLabel                   = "olm.owner"
```

Comment:
Could we consider importing the constant from [OLM package](https://github.com/operator-framework/operator-lifecycle-manager/blob/ad4ac77284454a077ecccfc6740f180bbb58e6f4/pkg/lib/ownerutil/util.go#L25)